hack/ci-utils/build.sh

```diff
@@ -1,7 +1,36 @@
 #!/bin/bash
 
-set -xe 
+set -xe
+if [[ -z "$1" ]]; then
+    go run hack/ci-utils/differenceChecker/main.go make generate
+    make build-all
+    make unit-test-go
+    exit 0
+fi
 
-go run hack/ci-utils/differenceChecker/main.go make generate
-make build-all
-make unit-test-go
+OPTION=$1
+case "$OPTION" in
+    "generate")
+        echo "Running make generate."
+        make generate
```

Comment:
maybe here you'd want to use the diff checker so you are sure that we have generated the files prior to pushing